### PR TITLE
Run tests during CI

### LINKED ISSUE #45: CI doesn't run tests
**Is this a BUG REPORT or FEATURE REQUEST?**:
bug

**What happened**:

I just realized that Travis wasn't actually running any tests, just building caddy with TXTDirect inside.

**What you expected to happen**:

Travis should run tests for TXTDirect in the "test" stage, as opposed to the "deploy" stage where building and uploading a Docker image is sufficient.

**How to reproduce it (as minimally and precisely as possible)**:

Trigger CI by opening a PR

### PR DESCRIPTION
**Pull request by alarsyo**
**What this PR does / why we need it**:

Previously we only tested if we could build correctly, now we also run tests.

**Which issue this PR fixes**
fixes #45

### CODE REVIEW

**Comment by alarsyo** [comment]
Woops, forgot that we needed to get the source before

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ services:
   - docker
 
 script:
+  - go test -v ./...
```
Comment (stp-ip) [inline .travis.yml]: This will most likely fail as the txtdirect plugin isn't yet linked. So it would need the linking as preparation step.

Comment (alarsyo) [reply .travis.yml]: I don't think the linking to caddy is actually needed for our tests, they run fine on their own

Comment (alarsyo) [reply .travis.yml]: At least they do on my machine

**Pull request merged**